src/styles/theme.scss

```diff
@@ -10,6 +10,11 @@ $white: #ffffff;
 $black: #000;
 $error: #fb5a5a;
 $black80: #333333;
+$lightBlue: #e8e5fb;
```

Comment:
we seem to already use `this-format` for color names rather than `thisFormat`, can we keep it consistent?